src/map/modifier.h

```diff
@@ -408,6 +408,8 @@ enum class Mod
     FERAL_HOWL_DURATION = 503,  // +20% duration per merit when wearing augmented Monster Jackcoat +2
     JUG_LEVEL_RANGE     = 564,  // Decreases the level range of spawned jug pets. Maxes out at 2.
     SIC_READY_RECAST    = 1052, // TODO: SIC/Ready recast reduction (seconds)
+    TANDEM_STRIKE       = 74,   // Beastmaster trait - provides acc/macc to master and pet when both engage the same target
+    TANDEM_BLOW         = 75,   // Beastmaster trait - provides subtle blow to master and pet when both engage the same target
```

Comment:
I would change these to something higher. TBH this file is so out of date with LSB when we get to a cherry-pick from them that modifies this file we are going have move some stuff around. Do you mind making an "ASB Section" at the bottom and starting at 2000 or something so when we do that it wont break anything?

Reply:
done